visallo/visallo
# rename concept to entity type

Testing Instructions:
- Make sure create new entity popover shows `Entity Type`
- Make sure histogram show `Entity Type`

Points of Regression: N/A

CHANGELOG
Changed: Rename concept to Entity Type
